slog_no121.go

```diff
@@ -0,0 +1,72 @@
+//go:build !go1.21
+// +build !go1.21
+
+package log
+
+import "time"
+
+type (
+	slogAttr struct {
+		Key   string
+		Value slogValue
+	}
+	slogValue struct{}
+	slogKind  int
+
+	slogLogValuer interface {
+		LogValue() slogValue
+	}
+)
+
+const (
+	slogKindAny slogKind = iota
+	slogKindBool
+	slogKindDuration
+	slogKindFloat64
+	slogKindInt64
+	slogKindString
+	slogKindTime
+	slogKindUint64
+	slogKindGroup
+	slogKindLogValuer
+)
```

Comment:
Here, we could simply use `golang.org/x/exp/slog`

Reply:
Great suggestion! :+1: